app/config.py

```diff
@@ -0,0 +1,35 @@
+import os
```

Comment:
może lepsza nazwa by była app-mode-config? I kolejna rzecz, czy dobrą praktyką jest robić kilka klas w 1 pliku? Może zamiast tego zrobić folder config, i tam to porozdzialać?

Reply:
Wydaje mi się że nie ma co wprowadzać mode do nazwy plików. Tak, w python siętak pisze że jest klika klas w jednym pliku. Tak samo będizemy robić  z kontrolerami. Chyba że będą bardzo złożone.

Reply:
Kilka kontrollwrów w 1 pliku brzmi jak strzal w kolano imo, co do nazwy kontrolleru to bym doprecyzowal, bo moze to byc po nazwie user config app config, db config, proxy co fig etc